sdk/iothub/azure-resourcemanager-iothub/CHANGELOG.md

```diff
@@ -1,14 +1,26 @@
 # Release History
 
-## 1.2.0-beta.3 (Unreleased)
+## 1.2.0-beta.3 (2023-04-18)
+
+- Azure Resource Manager IotHub client library for Java. This package contains Microsoft Azure SDK for IotHub Management SDK. Use this API to manage the IoT hubs in your Azure subscription. Package tag package-preview-2022-11. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
+
+
+
+* `models.IpVersion` was added
+
+
+
+* `ipVersion()` was added
+* `withIpVersion(models.IpVersion)` was added
 
 ### Features Added
 
-### Breaking Changes
+* `models.IpVersion` was added
 
-### Bugs Fixed
+#### `models.IotHubProperties` was modified
 
-### Other Changes
+* `ipVersion()` was added
```

Comment:
This part seems weird.

Reply:
It may happen when you run the script multiple times.. 

You may need to revert local changes before you run the script again.

Reply:
Logged an issue here, may or may not fix:
https://github.com/Azure/azure-sdk-for-java/issues/34529

Reply:
Fixed in the new version.